app/controllers/validators_controller.rb

```diff
@@ -25,14 +25,7 @@ def index
       admin_warning: validators_params[:admin_warning]
     )
 
-    @batch = Batch.last_scored(validators_params[:network])
-
-    if @batch
-      @this_epoch = EpochHistory.where(
-        network: validators_params[:network],
-        batch_uuid: @batch.uuid
-      ).first
-    end
+    set_batch_and_epoch
```

Comment:
perhaps this should be a before_action ?

Reply:
It could be. But on trent page, when there's no :q param, I don't want to query db for batch epoch unnecessarily. I'll set it as before_action on index only.